deptrac.php

```diff
@@ -9,7 +9,9 @@
 }
 
 (static function (): void {
-    require_once __DIR__.'/vendor/autoload.php';
+    if (\file_exists($autoload = __DIR__.'/vendor/autoload.php')) {
```

Comment:
To also support install as a global installed dependency there should be more paths taken into account, see https://github.com/sabre-io/dav/blob/c1afdc77a95efea6ee40c03c45f57c3c0c80ec22/bin/sabredav.php#L28

Reply:
I see what you mean, but this is already taken into the account. First of all, we do not have a `deprac/bin/deptrac`, but only "deptrac/deptrac". 

This is the difference that would make it so we have to go "up" one less directory (one less `../`).

If you take that into the account, then the path on line 12 is the same as in sabredev:29 and line 18 is the same as sabredev:30.

 - Why are they immediately called anonymous functions in deptrac? No idea.
 - Why are they separate? Again, no idea.
 
 Anyways this code works and is internally consistent. The only "improvement" I could see is that we could check if exactly one autoloader is found and if not, throw an error. But that is out of scope IMHO.

Reply:
Using the deptrac package directly is mainly a convenience feature for people working on it. Most people should still use the shim or phar. With that in mind, I think we don't have to make any more adjustments for now.